### ISSUE TITLE
Navigation bar not updating correctly when enabled/disabled

### ISSUE THREAD

**Issue by zooba** [Microsoft/PTVS#2678]
Reported at https://developercommunity.visualstudio.com/content/problem/68011/visual-studio-2017-classes-and-methods-barsdropdow.html

Seems we are not currently detecting the navigation bar being enabled via Tools/Options, and are not connecting correctly when editor windows are already open.

**Comment by athanhat** [comment]
​Hi, I experienced the same problem when I upgraded from Visual Studio 2015 to Visual Studio 2017. I have completely uninstalled the previous version of Visual Studio then I installed the new one with Python Anaconda environment. Visual Studio 2017 found two Python Environments a Python 3.6 (64-bit) that I have previously installed and the new Anaconda 4.3.0 but navigation dropdowns for classes and objects at the top of the editor and the object browser show nothing no matter which python environment I switch to.
